src/store/ducks/opensrp/hierarchies/index.tsx

```diff
@@ -128,6 +146,16 @@ export function fetchTree(
   };
 }
 
+export function fetchUpdatedCurrentParent(
+  currentParentId: string,
+  isRootJurisdiction: boolean
+): FetchUpdatedCurrentParent {
+  return {
+    currentParentId,
+    isRootJurisdiction,
+    type: FETCH_MAP_CURRENT_PARENT,
+  };
+}
```

Comment:
I am struggling to see why this is necessary.  It seems that we are using this as a way to keep the map's current parent in the redux store.  However, do we not already have that current parent i.e. when you are drilling up and down the JurisdictionTable component, you know the current parent at all times.  Why not just pass that to the map instead of doing this?

Reply:
The reason i added this action is that when drilling up and down using the map the we need to know what jurisdiction children need to be rendered next on the table/breacrumb since `currentParentId` will remain the same